wellcomecollection/content-api
# create endpoint to get venues

## What does this change?

https://github.com/wellcomecollection/content-api/issues/116
Adds `venues` to the content-api
can be queried by `id` or `title` eg. `/venues?title=library`


## How to test

Run locally and get `http://localhost:3000/venues?title=library`
Response should include regular opening dates, exceptional closures and the upcoming open days 

## How can we measure success?


## Have we considered potential risks?

Negligible, the API is not yet used by any of our services
